### DISCUSSION TITLE
Figuring out what is the most recent stable relaese

### DISCUSSION THREAD

**Discussion by Br0kenSilos** [TriliumNext/Trilium#4714] [Q&A]
I know this seems like a silly question. I like to keep my stuff as updated as possible for new features, fixes, etc.. But I remember seeing something not too terribly long ago about using the "latest" release is more like using a "beta" release. I have a server setup using docker and currently running 0.62.5. I know some folks had issues when they upgraded to 0.63.x with database corruption and sync issues. So I would like to upgrade the server and my Windows clients. But my questions is.. What is the most current "stable" release? Is 0.63.3 now considered the "stable" release?

**Comment by zadam** [comment]
Yes, 0.63.3 is the latest stable.

What you're referring to was a one time accident where a beta release was marked as stable.
